test/gpu/native/sharedMemory.skipif

```diff
@@ -0,0 +1 @@
+CHPL_GPU=CPU
```

Comment:
Does this need `==`? the others have `==`.

Reply:
Worse than that, this file looks like this file is cruft (there's no `sharedMemory.chpl` test in that dir there's one under `memory/` and in there I have `==`). I've removed this instance of `sharedMemory.skipif`.